pylint/checkers/similar.py

```diff
@@ -466,12 +466,12 @@ def get_map_data(self):
         """Passthru override"""
         return Similar.get_map_data(self)
 
-    @classmethod
-    def reduce_map_data(cls, linter, data):
+    def reduce_map_data(self, linter, data):
         """Reduces and recombines data into a format that we can report on
 
         The partner function of get_map_data()"""
         recombined = SimilarChecker(linter)
+        recombined.min_lines = self.min_lines  # Copy down relevant options
```

Comment:
By the way it is probably needed to add other options:
```
        recombined.ignore_comments = self.ignore_comments
        recombined.ignore_docstrings = self.ignore_docstrings
        recombined.ignore_imports = self.ignore_imports
        recombined.ignore_signatures = self.ignore_signatures
```

Reply:
@JulienPalard in the MR #4565 i had to add those lines in order to retrieve the same options in each process. But maybe i'm mistaken...

Reply:
As far as I understood, reduce_map_data is called only once from the "main" process to aggregate already collected lines from sub-processes. The ignore_* had already been taken care of by the subprocess: they just didn't gathered those lines.

Maybe in your new algorithms, the sub-process are ignoring ignore_*, and you handle them in the main process?

Reply:
Should we merge this MR and keep that in mind when we rebase on master for #4565 ? Or maybe add a test for ignore_comments, docstrings, imports or signatures consistency with multiprocessing right now to be sure ?

Reply:
@Pierre-Sassoulas i won't be able to investigate this before this weekend at best. So I think we could merge this as is. If @JulienPalard wants to add tests for ignore options it can be interesting but it is not mandatory for this PR which is "min-lines" oriented.

Reply:
I don't know when I'll have free time in the near future.

Anyway if #4565 pass the ignore_* parameter that may not be usefull it does not break anything, so I have nothing against passing them. It reduces surprise and break nothing: it's not bad (and could even become usefull in the future, who knows).